changelog.md

```diff
@@ -1,4 +1,12 @@
 # ACEmulator Change Log
+### 2017-04-29
+[Jyrus]
+* Changed the structure of the table that contains the Portal Destinations to include other portal properties
+* Renamed table from portal_destination to ace_portal_object and added a foreign key constraint to the weenieClassId field
+* !!! Please note that the ACE-World tables must be filled before importing the ace_portal_object's table / the Desert Mine sql
+	file contains an entry that is currently missing from the ACE-World DB release that also needs to be imported before the
+	ace_portal_object table is loaded.
+* Implemented a portal recall debug command - An apparent bug was revealed that causes players on login to start at the last saved portal recall position

```

Comment:
Bugs should also be reported in the Github issues section or noted in the bugged code section.